### PR TITLE
Register Subdomain: alisabry

### PR DESCRIPTION
**Pull request by ali-sabry**
Added the domain: alisabry

### CODE REVIEW

File: domains/alisabry.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "ali-sabry",
+        "email": "alisabry.official@gmail.com"
+      },
+
+      "record": {
+        "URL": "https://alisabry.is-a.dev"
```
Comment (WilliamDavidHarrison) [inline domains/alisabry.json:9]: You can not have your is-a.dev subdomain here as it will create a redirect loop. Please add valid records.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by ali-sabry — COMMENTED** [review]
Good Job!

**Comment by WilliamDavidHarrison** [comment]
> Good Job!

Why are you reviewing your own PR? It won't get it merged.

**Comment by WilliamDavidHarrison** [comment]
Read the review I made above.

File: domains/alisabry.json (hunk 1 above)
Comment (ali-sabry) [reply domains/alisabry.json:9]: How can I add a valid records

Comment (WilliamDavidHarrison) [reply domains/alisabry.json:9]: Read the [docs](https://is-a.dev/docs)

**Review by andrewstech — APPROVED** [review]

**Pull request merged**